index.html

```diff
@@ -502,6 +502,11 @@ <h3>Character</h3>
 &lt;/metadata&gt;
 ...
         </pre>
+            <li>All <code>ttm:agent</code> elements SHOULD be contained in a single <code>metadata</code> element.
```

Comment:
We could additionally recommend that every `<ttm:agent>` element's path SHOULD be `/tt/head/metadata[0]/ttm:agent`.

Reply:
I pushed a change to update this sentence to follow your recommendation.